www/js/views/server-setup.js

```diff
@@ -32,7 +32,11 @@ var ServerSetupView = Backbone.Marionette.ItemView.extend({
         var server = this.model;
         var self = this;
         var hostname = this.$('input[name=hostname]').val();
-        this.model.setup({hostname: hostname}, function (job) {
+        var encryption_enabled = this.$('input[name=encryption_enabled]');
+        this.model.setup({
+            hostname: hostname,
+            encryption_enabled: encryption_enabled.is(':checked') || false
```

Comment:
Pretty sure the 'is()' function will return either true or false, so the "|| false" is not needed.

Reply:
Very true.